bay_platform_dependencies.install

```diff
@@ -60,4 +60,25 @@ function bay_platform_dependencies_update_8001() {
       }
     }
   }
-}
\ No newline at end of file
+}
+
+/**
+ * Enable section purge module with new name change.
+ */
+function bay_platform_dependencies_update_10002() {
+  // Update section purger plugin settings.
+  $source_config = \Drupal::config('section_purger.settings.8714ff77fc');
+  $source_config_values = $source_config->get();
+  // Check if module is both installed and enabled.
+  if (\Drupal::moduleHandler()->moduleExists('section_purger')) {
+    // If exists, uninstall module.
+    \Drupal::service('module_installer')->uninstall(['section_purger']);
+  }
+  // Replace 'section_purger' with 'section_purge'.
+  $module_name = ['section_purge'];
+  \Drupal::service('module_installer')->install($module_name);
```

Comment:
Seems like a waste of code, just push the module name into the install function call? I can't see you using it further down?

Reply:
done, thanks